src/pages/MyPage/MyPage.js

```diff
@@ -1,7 +1,76 @@
 import React from 'react';
+import MypagePoints from './MypagePoints';
+import './MyPage.scss';
 
-const MyPage = () => {
-  return <div />;
+const Mypage = () => {
+  return (
+    <div className="MypageWrap">
+      <div className="imgUploadWrap">
+        <img
+          alt="defaultImg"
+          className="imgUploadDefault"
+          src="/images/mypage/logo@3x.png"
+        />
+        <div className="imgUploadBtn">
+          <img
+            alt="buttonImg"
+            className="imgUploadBtnImg"
+            src="/images/mypage/img_profile.png"
+          />
+        </div>
+      </div>
+      <div className="myPageInfoWrap">
+        <div className="myPageInfoWelcome">
+          <button type="button" className="gearBtn">
+            <img
+              alt="gearBtnImg"
+              className="gearBtnImg"
+              src="/images/mypage/btn_mypageSetting.jpeg"
+            />
+          </button>
+          <span className="myPageUserName">
+            <strong>유재민님,</strong>
+          </span>
+          <span className="myPageGreetings">환영합니다.</span>
+        </div>
+        <div className="MypageInformationLevel">
+          <p className="myPageLevel">DOCS FAMILY</p>
+        </div>
+        <div className="MypageSummaryItems">
+          {pointList.map(el => (
```

Comment:
```suggestion
          {pointList.map(({id, pointLabel, ~~~~~}) => (
```
인자가 객체인경우 매개변수 자리에서 바로 구조분해할당 가능합니다!